src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -2098,9 +2095,6 @@ internal override bool IsReadOnly
                 {
                     var isReadOnly = ThreeState.False;
 
-                    //PROTOTYPE(span): Span and ReadOnlySpan should have IsReadOnly attribute, eventually.
-                    //                 For now assume that any "System.Span" and "System.ReadOnlySpan" structs 
-                    //                 are ReadOnly
                     if (this.IsSpanType())
```

Comment:
>IsSpanType [](start = 29, length = 10)

About the implementation of this method, do we want to keep it the same? are we changing it before going to master?

Reply:
I think we still have versions of span that are not decorated. It was fixed very recently and may not yet propagated to various binaries/nugets
There is no harm with keeping this a little longer. I think we should merge with this.

---
In reply to: [138713521](https://github.com/dotnet/roslyn/pull/21982#discussion_r138713521) [](ancestors = 138713521)